hw/ip/hmac/data/hmac.hjson

```diff
@@ -374,6 +387,10 @@
           name: "fifo_depth",
           desc: "FIFO entry count."
         }
+        { bits: "10",
```

Comment:
Wouldn't it make more sense to put this status bit into the first nibble (i.e., at bit 2), together with the FIFO status bits?

Reply:
I would move it to bit 0 then and have all the FIFO relevant fields (empty, full and depth) in sequence afterwards. This will probably break a couple of DV checks, but all quickly fixed. How does that sound?